packages/contracts-bedrock/test/L2/GasPriceOracle.t.sol

```diff
@@ -8,23 +8,32 @@ contract GasPriceOracle_Test is CommonTest {
     event OverheadUpdated(uint256);
     event ScalarUpdated(uint256);
     event DecimalsUpdated(uint256);
-
     address depositor;
 
     // The initial L1 context values
     uint64 constant number = 10;
     uint64 constant timestamp = 11;
     uint256 constant basefee = 100;
+    uint256 constant blobBasefee = 101;
     bytes32 constant hash = bytes32(uint256(64));
     uint64 constant sequenceNumber = 0;
     bytes32 constant batcherHash = bytes32(uint256(777));
     uint256 constant l1FeeOverhead = 310;
     uint256 constant l1FeeScalar = 10;
+    uint32 constant blobBasefeeScalar = 15;
+    uint32 constant basefeeScalar = 20;
 
     /// @dev Sets up the test suite.
     function setUp() public virtual override {
         super.setUp();
+        depositor = l1Block.DEPOSITOR_ACCOUNT();
+    }
+}
 
+contract GasPriceOracleBedrock_Test is GasPriceOracle_Test {
+    /// @dev Sets up the test suite.
+    function setUp() public virtual override {
+        super.setUp();
         depositor = l1Block.DEPOSITOR_ACCOUNT();
```

Comment:
You can remove this redundant assignment of `depositor` since it happens in `super.setUp`

Reply:
done